composer.json

```diff
@@ -32,10 +32,9 @@
         "patches": {
             "drupal/core": {
                 "Color module html preview optional": "https://www.drupal.org/files/issues/color-optional-html-preview-2844190-2.patch",
-                "Restrict images to this site blocks image style derivatives": "https://www.drupal.org/files/issues/image_restrict_image_styles-2528214-24.patch",
+                "Restrict images to this site blocks image style derivatives": "https://www.drupal.org/files/issues/2018-06-14/2528214-36.patch",
```

Comment:
There is a newer patch available here: https://www.drupal.org/files/issues/2018-06-22/2528214-40.patch.

Reply:
Correct, but 40 is not compatible with Drupal 8.6. And 40 is not solving the issue completely as well. See my comment on this issue: https://github.com/goalgorilla/open_social/pull/976 I think it deserves more attention from ITcrowd as well. Especially if we want to get this in for certain enterprise projects.

Reply:
I thought that was indeed the case, but couldn't find anymore where it was said. Check.